sklearn/preprocessing/_encoders.py

```diff
@@ -448,8 +474,13 @@ def inverse_transform(self, X):
             n_transformed_features = sum(len(cats)
                                          for cats in self.categories_)
         else:
-            n_transformed_features = sum(len(cats) - 1
-                                         for cats in self.categories_)
+            if isinstance(self.drop, str) and self.drop == 'if_binary':
```

Comment:
Please use elif if it is appropriate here

Reply:
done